students/LuisGonzalez/session06/trapz.py

```diff
@@ -0,0 +1,11 @@
+def trapz(fun, a, b, N, **kwargs):
```

Comment:
if only args is used, they i would need to be careful on their position as to how i passed them to the function, and i wouldnot pass the arguments as A=A, instead it would only be the value/variable.
i prefer kwargs since because it makes more sense to me to handle the arguments as keyworded.
args would go infront of kwargs if both are used
i hope my understanding of args and kwards is correct. please let me know if missed anything.